### PR TITLE
[master-6.0] Fix CDO code generation customization for CDO XML TA

### PR DESCRIPTION
**Pull request by arnauddieumegard**

### CODE REVIEW

File: vpdsl/plugins/org.polarsys.capella.cdoxml.ta/src/org/polarsys/capella/cdoxml/ta/CapellaCDOandXMLTargetApplication.java [hunk 1]
```diff
@@ -12,18 +12,18 @@
...
 
 import org.polarsys.capella.cdoxml.ta.genchain.CapellaConnectorExtension;
+import org.polarsys.capella.extension.genchain.CapellaCdoGenerationExtension;
 import org.polarsys.capella.ta.definition.CapellaTargetApplication;
 
 import java.util.List;
 import org.eclipse.egf.portfolio.genchain.extension.ExtensionHelper;
-import org.eclipse.egf.portfolio.genchain.cdo.CdoGenerationExtension;
 
 public class CapellaCDOandXMLTargetApplication extends CapellaTargetApplication  {
```
Comment (bzendagui) [inline vpdsl/plugins/org.polarsys.capella.cdoxml.ta/src/org/polarsys/capella/cdoxml/ta/CapellaCDOandXMLTargetApplication.java:21]: I think it is better to change the super class to CDOForCapellaTargetApplication and remove the line 26.

**Review by bzendagui — CHANGES_REQUESTED** [review]

**Comment by pdulth** [comment]
Can you create a new issue for this CDO-XML as the #141 is closed and has a 5.2 release (already published)

File: vpdsl/plugins/org.polarsys.capella.cdoxml.ta/src/org/polarsys/capella/cdoxml/ta/CapellaCDOandXMLTargetApplication.java (hunk 1 above)
Comment (arnauddieumegard) [reply vpdsl/plugins/org.polarsys.capella.cdoxml.ta/src/org/polarsys/capella/cdoxml/ta/CapellaCDOandXMLTargetApplication.java:21]: You are right

**Comment by arnauddieumegard** [comment]
> Can you create a new issue for this CDO-XML as the #141 is closed and has a 5.2 release (already published)

Done in #149

**Comment by pdulth** [comment]
Merged on master-6.0

**Pull request closed without merging**